From 6db58c603805660216318d0d657665d845d84fdb Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@pulkomandy.tk>
Date: Tue, 19 Dec 2017 21:38:49 +0100
Subject: Fix detection of number of CPUs

For some reason awk says "Binary" which is not a number. Use wc.

diff --git a/common/autoconf/build-performance.m4 b/common/autoconf/build-performance.m4
index 4abbe4b..c8c7ad3 100644
--- a/common/autoconf/build-performance.m4
+++ b/common/autoconf/build-performance.m4
@@ -45,7 +45,7 @@ AC_DEFUN([BPERF_CHECK_CORES],
     NUM_CORES=`/usr/sbin/prtconf | grep "^Number Of Processors" | awk '{ print [$]4 }'`
     FOUND_CORES=yes
   elif test "x$OPENJDK_BUILD_OS" = xhaiku; then
-    NUM_CORES=`sysinfo -cpu | grep "running at" | awk '{print [$]1}'`
+    NUM_CORES=`sysinfo -cpu | grep "CPU" | wc -l`
     FOUND_CORES=yes
   elif test -n "$NUMBER_OF_PROCESSORS"; then
     # On windows, look in the env
-- 
2.15.0

